src/modules/jcms_migrate/src/EventSubscriber/JCMSMIgrateEventSubscriber.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drupal\jcms_migrate\EventSubscriber;
+
+use Drupal\entityqueue\Entity\EntitySubqueue;
+use Drupal\migrate\Event\MigrateEvents;
+use Drupal\migrate\Event\MigratePostRowSaveEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class JCMSMIgrateEventSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
+    $events[MigrateEvents::POST_ROW_SAVE] = 'onPostRowSave';
+    return $events;
+  }
+
+  /**
+   * Code to run after a row has been saved.
+   */
+  public function onPostRowSave(MigratePostRowSaveEvent $event) {
+    if ($event->getMigration()->id() == 'jcms_covers_db') {
+      if ($event->getRow()->getDestinationProperty('counter') <= 3) {
+        $subqueue = EntitySubqueue::load('covers');
+        $items = $subqueue->get('items')->getValue();
```

Comment:
For these sorts of things, I tend to be more defensive. Unless you're sure `getValue()` won't run on `null` I'd maybe separate this. It's not a huge issue though, if you're confident in it.